arctic/chunkstore/chunkstore.py

```diff
@@ -283,10 +284,13 @@ def write(self, symbol, item, chunker=DateChunker(), **kwargs):
             doc[METADATA] = {'columns': data[METADATA][COLUMNS] if COLUMNS in data[METADATA] else ''}
```

Comment:
I could do that. It would make it more robust going forward, but would not be backwards compatible. Might be worth thinking about. A breaking change now would be feasible, but months down the road will not be.